pkg/storaged/vdos-panel.jsx

```diff
@@ -28,7 +28,11 @@ const _ = cockpit.gettext;
 function vdo_row(client, vdo) {
     const block = client.slashdevs_block[vdo.dev];
     return {
-        client, key: vdo.name, kind: "array", name: vdo.name, devname: vdo.dev,
+        client,
+        key: vdo.name,
+        kind: "array",
+        name: vdo.name,
+        devname: vdo.dev,
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17890-20221107-162851-fa6168ae-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)